public/main.js

```diff
@@ -0,0 +1,32 @@
+const clickEvent = function (event) {
+    const id = event.target.getAttribute('data-question-id');
+    const helpful = JSON.parse(event.target.getAttribute('data-helpful'));
+
+    const options = {
+        method: "POST",
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+            id: id,
+            helpful: helpful
+        })
+    };
+    fetch("/rating", options)
+        .then(response => response.json())
```

Comment:
What's this line doing?

Reply:
the route "/rating" is the Url that I and Khalid communicate with the server 
it's the same like `url: "/api/helpful"`, it's our argument

Reply:
I mean specifically the line `.then(response => response.json())` but I think I understand now. That's converting the response string into an object, yeah?